examples/todo/src/application.ts

```diff
@@ -24,6 +27,14 @@ export class TodoListApplication extends BootMixin(
     // Set up default home page
     this.static('/', path.join(__dirname, '../public'));
 
+    // customize the swagger-ui
+    this.bind(RestExplorerBindings.CONFIG).to({
```

Comment:
Can we use `app.configure()` instead?

/cc @raymondfeng

Reply:
👍 refactored to be 
```ts
    this.configure(RestExplorerBindings.COMPONENT).to({
      swaggerThemeFile: '/theme-newspaper.css',
    });
```